src/After.tsx

```diff
@@ -52,6 +52,11 @@ class Afterparty extends React.Component<AfterpartyProps, AfterpartyState> {
         ...rest
       })
         .then(({ data }) => {
+          // if data is not for current location just don't do anything
+          if (location !== nextProps.location) {
```

Comment:
Isn't this condition always going to be falsy?

Reply:
yeap, thanks. 👍